Tools/clinic/clinic.py

```diff
@@ -4759,7 +4759,7 @@ def state_parameter(self, line):
                 self.parameter_state = self.ps_optional
             default = default.strip()
             bad = False
-            ast_input = "x = {}".format(default)
+            ast_input = f"x = {default}"
             bad = False
```

Comment:
Déjà vu... `bad = False` ... again :)

```suggestion
```

Reply:
presumably, it's just really, really bad